packages/expo-store-review/CHANGELOG.md

```diff
@@ -8,6 +8,12 @@
 
 ### 🐛 Bug fixes
 
+## 2.1.4 - 2020-08-07
+
+### 🎉 New features
+
+- [store-review] Implement [In-App Review](https://developer.android.com/guide/playcore/in-app-review) for Android
+
 ## 2.1.3 — 2020-07-27
 
```

Comment:
```suggestion
## 2.1.3 — 2020-07-27

### 🎉 New features

- [store-review] Implement [In-App Review](https://developer.android.com/guide/playcore/in-app-review) for Android
```

Reply:
this feature will go under v2.1.3? Wouldn't that be confusing? That version was already released

Reply:
Sorry, my bad. It should go under "Unpublished"

Reply:
moved to the unpublished section